server/src/controllers/AuthController.js

```diff
@@ -68,85 +80,111 @@ module.exports = {
 
       const user = await User.create(req.body);
 
-      if (process.env.NODE_ENV === "production") {
-        var createContact = new SibApiV3Sdk.CreateContact(); // CreateContact | Values to create a contact
-        createContact = { email: user.email };
-        apiInstance.createContact(createContact).then(
-          function (data) {
-            console.log("SIB contact created successfully.");
-          },
-          function (error) {
-            console.error(error);
-          }
-        );
-      }
-      const jwtUser = buildJWTPayload(user);
-      const token = jwt.sign({ user: jwtUser }, config.privateKey, {
-        expiresIn: ACCESS_TOKEN_EXPIRE_TIME,
-      });
-
-      const refreshToken = jwt.sign({user: jwtUser}, config.refreshPrivateKey, {
+			if (process.env.NODE_ENV === 'production') {
+				var createContact = new SibApiV3Sdk.CreateContact(); // CreateContact | Values to create a contact
+				createContact = { email: user.email };
+				apiInstance.createContact(createContact).then(
+					function (data) {
+						console.log('SIB contact created successfully.');
+					},
+					function (error) {
+						console.error(error);
+					}
+				);
+			}
+			const jwtUser = buildJWTPayload(user);
+			const token = jwt.sign({ user: jwtUser }, config.privateKey, {
+				expiresIn: ACCESS_TOKEN_EXPIRE_TIME,
+			});
+
+			const refreshToken = jwt.sign({user: jwtUser}, config.refreshPrivateKey, {
```

Comment:
`token` and `refreshToken` will not be required here anymore, so we can remove them.